snappy/dpu_snappy.c

```diff
@@ -210,8 +211,8 @@ int main(int argc, char **argv)
 			write_output_host(output_file, &output);
 
 		if (compress) {
-			printf("Compressed %ld bytes to: %s\n", output.length, output_file);
-			printf("Compression ratio: %f\n", 1 - (double)output.length / (double)input.length);
+			// printf("Compressed %ld bytes to: %s\n", output.length, output_file);
```

Comment:
If you don't want these to be printed every time, add them to an #ifdef DEBUG section